RealmSwift/Aliases.swift

```diff
@@ -81,9 +81,9 @@ extension ObjectBase {
      provided key paths. For example, if:
      ```swift
      class Dog: Object {
-         @objc dynamic var name: String = ""
-         @objc dynamic var adopted: Bool = false
-         let siblings = List<Dog>()
+         @Persisted var name: String = ""
```

Comment:
These also don't need default values anymore. This goes for all of the fixed docstrings.